### DISCUSSION TITLE
定时调度

### DISCUSSION THREAD

**Discussion by obobj** [apache/doris#6630] [A - General / Q&A] [answered]
请问目前doris支持像MySQL一样的定时调度功能么，如果这样我们能够把etl的操作都放在doris完成，直接微批的生成数仓的表了

**Comment by liutang123** [comment, marked as answer]
Doris目前没有类似的功能

**Comment by hf200012** [comment]
You can use the dolphin scheduler :https://dolphinscheduler.apache.org/
